src/components/irregularity/helpers/Irregularity.styled.tsx

```diff
@@ -0,0 +1,84 @@
+import { styled } from '@mui/material/styles'
+import { Button, Grid } from '@mui/material'
+
+import LinkButton from 'components/common/LinkButton'
+import SubmitButton from 'components/common/form/SubmitButton'
+import theme from 'common/theme'
+
+export const Root = styled(Grid)(() => ({
+  marginTop: theme.spacing(15),
+  textAlign: 'center',
+}))
+
+export const Icon = styled(Grid)(() => ({
+  justifyContent: 'center',
+  marginBottom: theme.spacing(4),
+
+  '& svg': {
+    fontSize: theme.typography.pxToRem(120),
+  },
+}))
+
+export const ButtonsWrapper = styled(Grid)(() => ({
+  justifyContent: 'space-evenly',
+}))
+
+export const ActionLinkButton = styled(LinkButton)(() => ({
+  backgroundColor: '#0098E3',
+  border: '1px solid #909090',
+  padding: theme.spacing(1, 5),
+  borderRadius: theme.borders.round,
+  color: theme.palette.common.white,
+  fontSize: theme.typography.pxToRem(18),
+
+  '&:hover': {
+    backgroundColor: '#62C4FB',
+    color: theme.palette.common.black,
+    border: '1px solid #909090',
+  },
```

Comment:
Shouldn't that `LinkButton` add the hover automatically?

Reply:
It does add a hover effect but it is not as the design suggests. Also it adds wider borderRadius and the button becomes bigger on hover which is not ok.